### ISSUE TITLE
Final Project Peer Review

### ISSUE THREAD

**Issue by mdeledebur** [zxhuang96/used_car_pricing#18]
This project develops a model to determine the appropriate price of a used car for Craigslist Listings. They are using a sales dataset from Craigslist. Their objective is to develop an accurate model that can be used by both buyers and sellers of used cars on Craigslist.

Three things I liked:
- Good descriptions of data transformation and good job including other relevant data from outside their dataset into their model and dropping irrelevant features from the original dataset.
- Good job of avoiding overfitting in their models
- Good job of fine tuning parameters in each of their models

Three areas for improvement:
- For features with NAN values greater than 50%, instead of dropping them, try imputing the values because those features could be important!
- Could use some more exploratory data analysis before fitting models
- Did not explain reasoning behind the loss functions they used in each model and did not test different kinds of loss functions to see if it could improve their model.
